runtime/testing-basilisk/src/lib.rs

```diff
@@ -903,7 +909,7 @@ construct_runtime!(
 		Marketplace: pallet_marketplace::{Pallet, Call, Event<T>, Storage} = 109,
 
 		// ORML related modules - starts at 150
-		Currencies: orml_currencies::{Pallet, Call, Event<T>} = 150,
+		Currencies: orml_currencies::{Pallet, Call} = 150,
```

Comment:
Currencies don't fire events anymore?

Reply:
No. It's not necessary because pallets for native and non-native currencies handle this.

Reply:
i am still using events from `Currencies` pallet, pls don't remove it

Reply:
Doesn't FRAME have a shorthand now? (`Currencies: orml_currencies = 150`)

Reply:
yes, thanks. @lumir-mrkva The events from `Currencies` have been removed. So it's not possible to keep them.

Reply:
not sure how we will track balance transfers currently consitently, we should probably add them to the adapter then

Reply:
this is really breaking change for us

Reply:
#545 this PR replaces `Currencies` with our own version of the pallet (latest version with the events).